kaiju/Tabs.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Tabs",
+  "library": "terra-tabs",
+  "display": "Tabs",
+  "description": "Container for organizing content",
+  "group": "Molecules::Tabs",
+  "properties": {
+    "tabFill": {
+      "type": "Bool"
```

Comment:
can we add a display to tabFill and fill properties to describe what is different between them, I think users may not understand the difference

Reply:
This point was brought up with our UX designers. We agreed that it's more important to match the API of the component to facilitate conversation with engineers. We also think we may want to go back and deprecate the tabFill prop in favor of an isExtended prop in tabs itself.